packages/neuron-wallet/package.json

```diff
@@ -13,18 +13,17 @@
   "main": "dist/main.js",
   "license": "MIT",
   "scripts": {
-    "start": "tsc && electron .",
+    "start": "cross-env NODE_ENV=production electron .",
+    "start:dev": "tsc && cross-env NODE_ENV=production electron .",
```

Comment:
We've used `isPackaged` to distinguish build for dev or prod.

Reply:
Hold on a second, will update this.